src/OpenFeature/Providers/MultiProvider/Models/ProviderStatus.cs

```diff
@@ -0,0 +1,7 @@
+namespace OpenFeature.Providers.MultiProvider.Models;
+
+internal class ProviderStatus
+{
+    public string ProviderName { get; set; } = string.Empty;
+    public Exception? Exception { get; set; }
```

Comment:
I would suggest to use a more descriptive name, such as `Error` or `ExceptionDetail`, to clearly indicate the purpose of the property and avoid shadowing. Reusing the type name as a property name can be confusing.